### ISSUE TITLE
Obsbot Tail PTZ Camera

### ISSUE THREAD

**Issue by justibro** [bitfocus/companion-module-requests#567]
The device is here https://www.obsbot.com/obsbot-tail 

It has an api that access over http for basic control.  
[OBSBOT Tail HTTP API Document.pdf](https://github.com/bitfocus/companion-module-requests/files/6909115/OBSBOT.Tail.HTTP.API.Document.pdf)
Someone else also found this info to control the position:
-----------
POST request of JSON data
{
"cmd" : "setAbsDegree",
"rollDegree" : roll,
"pitchDegree" : pitch,
"yawDegree" : yaw
}
to endpoint: http://192.168.0.1:27739/obsbot/tail/ai/gimbal
will make your Obsbot Tail rotate. Use an invalid value such as -1000 if you don't want to update certain angles.
-----------
What would be nice to have is the ability to control most of the setting and have the feedback of position.
What is lacking from their own app is the ability to set and recall presets which is in the api.

I have created my own website interface to control the camera but it would be nice to use a button interface to move it around as-well.

**Comment by chrisspiegl** [comment]
Alternatively, the OBSBOT Tail Air also understands OSC commands. Maybe that's an option to implement features which the Tail Air does not provide via the Visca compatible layer?

Documentation for OSC Commands on the OBSBOT Tail Air: https://www.obsbot.com/explore/obsbot-center/osc

**Comment by Lexorius** [comment]
Hi, the obsbot Tail Air understands the sony visca module.  for most things the sony visca module should work. 

 https://www.obsbot.com/de/explore/obsbot-tail-air/visca-over-ip

but it would be nice to have a komplete integration in companion.

**Comment by chrisspiegl** [comment]
Absolutely, the VISCA over IP is already a great help. However, there are a lot of nice features that are only available over the other connection methods. Setting AI Tracking modes, start/stop recording, set the exposure compensation (they appear to not stay true to the VISCA commands on that point).

Maybe a "add-on" companion module would be an idea so that it can be used next to the VISCA module? No need to double the functionality, IMO.
